src/patternfly/components/Page/page.scss

```diff
@@ -475,6 +475,10 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
   flex-shrink: 0;
 }
 
-.#{$page}__drawer > .#{$drawer} {
-    flex: 1 0 auto;
+.#{$page}__drawer .#{$page}__main-container {
```

Comment:
There should only be one `.#{$page}__main-container` in `.#{$page}__drawer` unless anyone can think of a situation where that isn't the case? cc @srambach @mattnolting @dlabrecq